frontend/js/utils/battle-detail.js

```diff
@@ -0,0 +1,14 @@
+const showTeams = (battle, user) => {
+  let currentUserTeam = null;
+  let opponentUserTeam = null;
+  currentUserTeam =
+    battle.teams[0].trainer.email === user.email ? battle.teams[0] : battle.teams[1];
```

Comment:
What happens if the battle doesn't have teams?

Reply:
The teams are created with the battle, but I'll make a validation for the case :D